src/Components/Reports/Util.js

```diff
@@ -14,7 +14,7 @@ export const formatData = (data, type) => {
             let rowValue;
             if (rowKey === 'groups') {
                 rowValue =  get(systemItem, rowKey, []);
-                rowValue = rowValue.length === 0  ? 'N/A' : rowValue[0].name;
+                rowValue = rowValue.length === 0  ? 'No group' : rowValue[0].name;
```

Comment:
Doesn't it miss div container with the `pf-v5-u-disabled-color-200` classname?

Reply:
This is just a pdf report. Not the table. The ticket describes changing N/A to No group, but to be consistent I wanted to change it here as well. There are no disabled text on the PDF

Reply:
@adonispuente I went through card and there is a method `displayGroup` in `RenderColumns.js` file which renders the group name or NA. We will need to update that as well right? Let me know if you have any questions or need my inputs.

Reply:
great catch, I'l push that up now !